packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.scss

```diff
@@ -153,64 +173,71 @@
   align-items: center;
 }
 
-.actionsOneline {
-  position: relative;
-}
-
 .dismissal {
   @include focus-border();
   min-width: 0;
-}
-.dismissalOneline .dismissal {
-  @include margin-right(-8px);
+  position: absolute;
+  top: 0;
+  @include right(0);
 }
 
-// Add space between adjacent MessageBars.
-.root + .root {
-  margin-top: 6px;
+.ms-MessageBar-dismissalOneline {
```

Comment:
we don't use global classes any more. This should be a camelCased name and referenced in the tsx using styles.{name}.